AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1134,6 +1134,9 @@ bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.
 semantica.in##.sideSocialSubscribe
 semantica.in##.sideSubscribe
 politros.com##.email-dispatch
+washingtonpost.com#?#aside > div[class]:has(> div > form#newsletter-form)
```

Comment:
I don't see this code in HTML. And as I remember, it can break page.

Reply:
<details><summary>Screenshot:</summary>

![5](https://user-images.githubusercontent.com/33488229/66235012-c650b780-e6f7-11e9-8524-5490b4f6d1ba.png)
</details><br/>

Reply:
still don't see. Does not it break the page like month ago?(compare in two tabs)

https://www.washingtonpost.com/graphics/2019/national/climate-environment/climate-change-siberia/?hpid=hp_hp-visual-stories-desktop_no-name%3Ahomepage%2Fstory-ans
missed